src/components/RecordTypesSelect/components/CompareRecordSelect.js

```diff
@@ -148,21 +153,23 @@ export const CompareRecordSelect = memo(({
             style={{
               width: compareElemWidth,
               height: `calc(${height}px + ${css.cellPadding} - ${css.selectedLineBorderWidth} - ${css.selectedLineBorderWidth})`,
+              right: IS_LOCALE_LTR ? 0 : null,
+              left: !IS_LOCALE_LTR ? `-${compareElemWidth}px` : null,
             }}
           >
-            <Triangle />
+            <Triangle direction={IS_LOCALE_LTR ? 'right' : 'left'} />
             <span className={css.compareMessage}>
               <FormattedMessage id="ui-data-import.compare" />
             </span>
-            <Triangle direction="left" />
+            <Triangle direction={IS_LOCALE_LTR ? 'left' : 'right'} />
           </div>
         </Col>
         <Col
           xs
-          className={classNames(css.cell, css.borderLeft)}
+          className={classNames(css.cell, { [css.borderLeft]: IS_LOCALE_LTR }, { [css.borderRight]: !IS_LOCALE_LTR })}
```

Comment:
You have a lot of dublicated part of code
try to cover them in separated variables

for example `const colStyles = { [css.borderLeft]: IS_LOCALE_LTR }, { [css.borderRight]: !IS_LOCALE_LTR }`

Reply:
But you suggest assigning to a variable 2 objects, separated by a comma.

Reply:
You are right, my mistake